R/validations.R

```diff
@@ -1,15 +1,16 @@
 #' Validate that dataset has a minimum number of observations
 #'
-#' @description `r lifecycle::badge("stable")`
-#' @param x a data.frame
+#' `r lifecycle::badge("stable")`
```

Comment:
Won't this push the second paragraph to `@details` section?

```suggestion
#' @description `r lifecycle::badge("stable")`
```

Reply:
all of the cases contain the same details i.e 
`This function is a wrapper for 'shiny::validate'.` It's better if we keep it under the @details tag.